seqr/views/utils/dataset_utils.py

```diff
@@ -111,39 +111,41 @@ def _find_or_create_missing_sample_records(
         samples,
         projects,
         user,
-        sample_ids,
+        sample_project_tuples,
+        get_individual_sample_key,
         raise_no_match_error=False,
         raise_unmatched_error_template=None,
         create_active=False,
         get_individual_sample_lookup=_get_individual_sample_lookup,
-        get_individual_sample_key=None,
```

Comment:
Change the `get_individual_sample_key` from an optional argument to a required one.